tests/python/unittest/test_init.py

```diff
@@ -67,10 +67,23 @@ def test_bilinear_init():
     bili_1d = np.array([[1/float(4), 3/float(4), 3/float(4), 1/float(4)]])
     bili_2d = bili_1d * np.transpose(bili_1d)
     assert (bili_2d == bili_weight.asnumpy()).all()
-    
+
+def test_const_init_dumps():
+    # test NDArray input
+    init = mx.init.Constant(mx.nd.ones(5))
```

Comment:
in line 29 `test_variable_init` and other initializer tests are written differently. can we follow a similar pattern of declaring a sym variable and binding it a module

Reply:
I am just checking the function of dumps() in the test I have added, is creating a module still required? `test_rsp_const_init` in line 48 already does that for Constant initializers.

Reply:
`test_rsp_const_init` does the following -
https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_init.py#L52-L57

The reason I am asking this change is because I think that will be a more comprehensive test and will check the whole flow, and it will test the initializer the way it will actually be used in production. This change should enable serializing and deserializing an `ndarray` object. 

Just check once with a committer who will merge this code. Everything else LGTM.